src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square implements Figure {
```

Comment:
Consider using a more descriptive name for the `Square` class. While `Square` is not a verb and doesn't violate the 'Don't use verbs for class/interface names' rule directly, it's often beneficial to have names that reflect the object's purpose or behavior, such as `DrawableSquare` if it's meant to represent a square that can be drawn.